pkg/apiserver/registry/intelligence/networkpolicyrecommendation/rest.go

```diff
@@ -47,44 +53,32 @@ func (r *REST) New() runtime.Object {
 	return &intelligence.NetworkPolicyRecommendation{}
 }
 
-func (r *REST) getNetworkPolicyRecommendation(name string) *intelligence.NetworkPolicyRecommendation {
-	npReco, err := r.npRecommendationQuerier.GetNetworkPolicyRecommendation("flow-visibility", name)
+func (r *REST) Get(_ context.Context, name string, _ *metav1.GetOptions) (runtime.Object, error) {
```

Comment:
nit: not sure if we should follow convention in Antrea and keep the unused arg name rather than using `_`, so that when later we add code using them, the arg names will be naturally kept same as the interface value.

Reply:
Thanks Shawn for reviewing. I've changed it to follow the style on Antrea side.